### PR TITLE
Correct download url and README filename error

### PR DESCRIPTION
**Pull request by CorcovadoMing**
Correct download url and README filename error

### CODE REVIEW

File: scripts/common.sh [hunk 1]
```diff
@@ -7,12 +7,12 @@ HADOOP_PREFIX=/usr/local/hadoop
 HADOOP_CONF=$HADOOP_PREFIX/etc/hadoop
 HADOOP_VERSION=hadoop-2.6.0
 HADOOP_ARCHIVE=$HADOOP_VERSION.tar.gz
-HADOOP_MIRROR_DOWNLOAD=http://archive.apache.org/dist/hadoop/core/hadoop-2.4.1/hadoop-2.4.1.tar.gz
+HADOOP_MIRROR_DOWNLOAD=http://archive.apache.org/dist/hadoop/core/hadoop-2.6.0/hadoop-2.6.0.tar.gz
```
Comment (lazyval) [inline scripts/common.sh]: Are you ok to not bump version?

Comment (CorcovadoMing) [inline scripts/common.sh]: @lazyval  I sent this PR to the branch "vangj:hadoop-2.6.0-spark-1.3.0"
It should use the 2.6.0 version of hadoop, or the installation failed

Comment (lazyval) [inline scripts/common.sh]: oh, I see

**Comment by lazyval** [comment]
:+1:

**Pull request merged**